src/flb_sds.c

```diff
@@ -389,6 +397,6 @@ void flb_sds_destroy(flb_sds_t s)
         return;
     }
 
-    head = FLB_SDS_HEADER(s);
```

Comment:
pls restore the original indentantion level

Reply:
Restored the original indentation level. (And did a force push because don't want to have one extra commit)